core/templates/pages/exploration-editor-page/exploration-editor-page.directive.html

```diff
@@ -15,7 +15,7 @@
     <translation-tab ng-if="$ctrl.getActiveTabName() === 'translation'">
     </translation-tab>
     <!-- This is an ng-if, so that the preview loads the latest version of the exploration each time the tab is accessed. -->
-    <preview-tab ng-if="$ctrl.getActiveTabName() === 'preview'">
+    <preview-tab ng-if="$ctrl.getActiveTabName() === 'preview'" class="protractor-preview-tab">
```

Comment:
Let's change the class name to `protractor-test-preview-tab` to fit the `protractor-test-*` convention we follow elsewhere

```suggestion
    <preview-tab ng-if="$ctrl.getActiveTabName() === 'preview'" class="protractor-test-preview-tab">
```

Reply:
Done.